analytics/terraform/spark-k8s-operator/addons.tf

```diff
@@ -93,9 +96,8 @@ module "eks_data_addons" {
         limits:
           cpu: 1000
         disruption:
-          consolidationPolicy: WhenEmpty
-          consolidateAfter: 30s
-          expireAfter: 720h
+          consolidationPolicy: WhenEmptyOrUnderutilized
+          consolidateAfter: 5m
```

Comment:
I see some NodePools with `1m` and some with `5m`, did we want these to be consistent? `5m` seems a bit long but I think I've used the default `30s` for a while

Reply:
I will update that